digital_land/commands.py

```diff
@@ -282,6 +293,9 @@ def pipeline_run(
     issue_log = duplicate_reference_check(issues=issue_log, csv_path=output_path)
 
     issue_log.save(os.path.join(issue_dir, resource + ".csv"))
+    operational_issue_log.save(
+        operational_issue_dir=operational_issue_dir, performance_dir=performance_dir
```

Comment:
maybe just call this output_dir in the save command command. then a directory can be passed in.